.travis.yml

```diff
@@ -26,7 +42,7 @@ before_install:
   - composer global require bamarni/composer-bin-plugin:dev-master
 
 install:
-  - composer install
+  - composer update --no-interaction
```

Comment:
This has changed because running composer install without a lock file is deprecated, and composer 2 will start complaining loudly about this.